balarco/settings/base.py

```diff
@@ -165,3 +165,6 @@
 
 STATIC_URL = '/static/'
 STATIC_ROOT = os.path.join(BASE_DIR, "../static/")
+
+MEDIA_URL = '/media/'
```

Comment:
Add the contents of this folder or the folders inside this folder to the gitignore, you are pushin images to the REPO...